compiler/noirc_evaluator/src/acir/tests/mod.rs

```diff
@@ -926,3 +927,208 @@ fn do_not_overflow_with_constant_constrain_neq() {
     assert_eq!(acir_functions.len(), 1);
     assert!(acir_functions[0].opcodes().is_empty());
 }
+
+// Convert the SSA input into ACIR and use ACVM to execute it
+// Returns the ACVM execution status and the value of the 'output' witness value,
+// unless the provided output is None or the ACVM fails during execution.
```

Comment:
```suggestion
/// Convert the SSA input into ACIR and use ACVM to execute it
/// Returns the ACVM execution status and the value of the 'output' witness value,
/// unless the provided output is None or the ACVM fails during execution.
```
nit: might as well make these doc comments if they are on the function